lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -651,17 +636,13 @@ defmodule RealtimeWeb.RealtimeChannel do
     })
   end
 
-  defp presence_list_shard_bypass(%{assigns: %{tenant_topic: topic}} = _socket) do
-    try do
-      [{:pool_size, size}] = :ets.lookup(Presence, :pool_size)
+  defp presence_dirty_list(topic) do
```

Comment:
it would be useful for debugging
```suggestion
  def presence_dirty_list(topic) do
```